recipes/node-semver/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "node-semver" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "c32bfc976fd9e003c4a15665e5fe9f337366ba6b60aeb34e4479da9d7bbb0081" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - semver
+
+about:
+  home: https://github.com/podhmo/python-semver
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: port of node-semver
+  dev_url: https://github.com/podhmo/python-semver
+
+extra:
+  recipe-maintainers:
+    - windelbouwman
```

Comment:
Is this also you? Why the two IDs?

Reply:
Yes that is also me. The two accounts are redundant.